packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -270,6 +270,7 @@ export const CollectionScreenDetailsTile = ({
       description={description}
       descriptionLinkPressSource='collection page'
       details={details}
+      hideOverflow={detailsTileProps.hideOverflow || !isReachable}
```

Comment:
Just confirming - DetailsTile had this prop but we weren't using it?

Reply:
Also do we want to hide it or disable it? Maybe a q for Julian, but maybe too specific at this point...